images/arrow_down.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Nice use of SVGs - we haven't really gone into image types but SVG is growing in popularity and a great tool to know and use when building websites.